### PR TITLE
Tried to make resize flavor tests easier to run.

### PR DESCRIPTION
**Pull request by TimSimpsonR**
- Added another flavor, this one with 768, we can use with the resize flavor test.
- Took out some code that raised SkipTest if something in the test config was turned off. Not sure why this wasn't caught, but this code (by nature of Proboscis) skips all other tests if set.
- Changed the resize flavor test code to use flavors from the config file (before it had literal flavor ids in the test code).

### CODE REVIEW

File: tests/integration/tests/api/instances_actions.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Copyright 2011 OpenStack LLC.
+\# Copyright 2011 OpenStack LLC.
```
Comment (cp16net) [inline tests/integration/tests/api/instances_actions.py]: whoops \

Comment (TimSimpsonR) [inline tests/integration/tests/api/instances_actions.py]: Fixed.

**Comment by imsplitbit** [comment]
I'm sorry but I'm missing the reason why we need a flavor that doesn't fit into our calculations for unit size.  Unit size is 512M and 768 breaks that as we decided we weren't going to offer fractional units ATM.

**Comment by cp16net** [comment]
@imsplitbit so this is just for the tests and more specifically the fake tests.

**Comment by cp16net** [comment]
+1

**Comment by TimSimpsonR** [comment]
Actually its for the real tests. They'd been resizing from 512mb to 2gb which was past the limit of my VM. That's why I made the flavor size configurable via the test conf so we don't have to always test against this weird flavor size of 768 except in our VM environments.

**Comment by imsplitbit** [comment]
Ok then +1

**Comment by rnirmal** [comment]
+1

**Comment by sacharya** [comment]
+1

**Comment by ed-** [comment]
+1

**Pull request merged**